p2p/discover/table.go

```diff
@@ -481,10 +489,18 @@ func (tab *Table) bucketAtDistance(d int) *bucket {
 //
 // The caller must not hold tab.mutex.
 func (tab *Table) addSeenNode(n *node) {
+	go tab.addSeenNodeSync(n)
```

Comment:
This can be a DOS vector since we just spawn new goroutines without max threshold